ofrak_core/ofrak/model/tag_model.py

```diff
@@ -64,8 +65,10 @@ def sort_tags_into_tiers(
 
         return tuple(tuple(level) for level in reversed(levels) if len(level) > 0)
 
+    @classmethod
+    @abstractmethod
     def caption(cls, attributes) -> str:
-        return str(cls.__name__)
+        raise NotImplementedError()
```

Comment:
Done.

FYI, I had initially decided to make `ResourceTag.caption` an abstract method because nothing relies on its implementation and it is a lot to step through `ResourceView -> ResourceViewInterface -> ViewableResourceTag -> ResourceTag` to find that implementation.

Reply:
It turns out its not possible to remove `ResourceView.caption`, since other caption implementations that call `super()` fail because `ResourceTag` isn't a superclass.

I've added reverted the `ResourceTag.caption` change and added comments to exclude that function from coverage (since it will never get called). If you want to change it back to an abstractmethod I'd support that as well, feel free to make that change.